13.go

```diff
@@ -122,7 +118,6 @@ type kexX25519 struct{ defaultServerKEX }   // Used by X25519
 type kexSIDHp503 struct{ defaultServerKEX } // Used by SIDH/P503
 type kexSIKEp503 struct{}                   // Used by SIKE/P503
 type kexHybridSIDHp503X25519 struct {
-	defaultServerKEX
```

Comment:
The default implementation was not sufficient as the client and server generates keys and a shared secret using different computations. Hence the specialization below.